tests/_get_binary_test.ts

```diff
@@ -28,3 +33,58 @@ Deno.test("Test download", async () => {
   await page.waitForSelector("h1");
   await browser.close();
 });
+
+{
+  const temp = await Deno.makeTempDir({
+    prefix: "astral_test_download_permissions",
+  });
```

Comment:
Is there a reason this is outside the test?

Reply:
Since the aim of this test is to check that it still work with restrained permissions (only write/read access to the specified dir), we need to know the path before executing the test so we can use `Deno.test` with the correct testing context

The condensed code below may help visualize this better:
```ts
const temp = await Deno.makeTempDir()
Deno.test("...", { permissions: { write: [temp], read: [temp], net: true } })

Reply:
Ah, sorry, I missed the point of the test. This makes sense to me 👍